wandb/sdk/data_types/object_3d.py

```diff
@@ -161,6 +218,54 @@ def __init__(
         else:
             raise ValueError("data must be a numpy array, dict or a file object")
 
+    @classmethod
+    def from_file(
+        cls, data_or_path: Union["TextIO", str], file_type: "FileFormat3D" = None
+    ) -> "Object3D":
+
+        return cls(data_or_path, file_type=file_type)
+
+    @classmethod
+    def from_numpy(cls, data: "np.ndarray") -> "Object3D":
+        if not util.is_numpy_array(data):
+            raise ValueError("`data` must be a numpy array")
+
+        if len(data.shape) != 2 or data.shape[1] not in {3, 4, 6}:
+            raise ValueError(
+                """The shape of the numpy array must be one of either
+                                [[x y z],       ...] nx3
+                                 [x y z c],     ...] nx4 where c is a category with supported range [1, 14]
+                                 [x y z r g b], ...] nx4 where is rgb is color"""
+            )
+
+        return cls(data)
+
+    @classmethod
+    def from_point_cloud(
+        cls,
+        points: Sequence["Point"],
+        boxes: Sequence["Box3D"],
+        vectors: Optional[Sequence["Vector3D"]] = None,
+        point_cloud_type: "PointCloudType" = "lidar/beta",
+        # camera: Optional[Camera] = None,
```

Comment:
why is this commented out?

Reply:
This will go in as a part of the next PR, just didn't make the finish line for this one.